Plugins/Podio/CMakeLists.txt

```diff
@@ -34,9 +34,13 @@ target_link_libraries(ActsPluginPodio PUBLIC
   podio::podioRootIO
 )
 
+target_compile_options(ActsPodioEdm PRIVATE "-Wno-old-style-cast")
```

Comment:
We have no control over this generated code, and it would be a pain to break it out of the build.